homeworks/aleksey.poddubnii_AlekseyPoddubnii/homework_6/user.js

```diff
@@ -0,0 +1,34 @@
+
+const user = {};
+
+Object.defineProperty(user, 'name', {
+    set: (input) => {
+        this.name = input.toLowerCase().trim().replace(/\b\w/g, name => name.toUpperCase());
+    },
+    get: () => this.name,
+});
+
+const plus = '+';
+
+Object.defineProperty(user, 'phone', {
+    set: (phone) => {
+        if (phone[0] === '+') {
+            this.phone = plus + phone.split(/\D/).join('');
+        } else {
+            this.phone = phone.split(/\D/).join('');
+        }
```

Comment:
`get` and `set` arrow functions as well... also try not to write exactly same code multiple times, if `phone.split(/\D/).join('')` operation is used in both cases, so maybe you should write it only once and add `+` if it was present initially

Reply:
DRY)) ok, try make better